### DISCUSSION TITLE
I don't understand this: "Once the web UI has been submitted"

### DISCUSSION THREAD

**Discussion by NathanWailes** [dokku/dokku#7091] [Q&A] [answered]
Hi everyone,

I followed [the guide to setting up Dokku for Digital Ocean][1] and everything seemed to go smoothly, but I don't understand this very last step in the guide:

> Once the web UI has been submitted, you will be redirected to our [application deployment tutorial](https://dokku.com/docs/deployment/application-deployment/), which will guide you through deploying a sample application to your Dokku server.

What is the web UI, and what does it mean for it to have been submitted?

[1]: https://dokku.com/docs/getting-started/install/digitalocean/

**Comment by NathanWailes** [comment]
Oh I think it's referring to the Digital Ocean UI for creating the new droplet, is that right?

**Comment by josegonzalez** [comment, marked as answer]
The web ui it was referencing - which was used for just setting up an SSH key and a domain - no longer exists. It was removed as it was not secured and lots of folks accidentally never used that to enter in their data, causing security breaches.
